src/components/Attachments/AttachmentCarousel/CarouselActions.tsx

```diff
@@ -1,25 +1,18 @@
-import lodashGet from 'lodash/get';
-import PropTypes from 'prop-types';
 import {useEffect} from 'react';
 import KeyboardShortcut from '@libs/KeyboardShortcut';
 import CONST from '@src/CONST';
 
-const propTypes = {
+type CarouselActionsProps = {
     /** Callback to cycle through attachments */
-    onCycleThroughAttachments: PropTypes.func.isRequired,
+    onCycleThroughAttachments: (deltaSlide: number) => void;
 };
 
-function CarouselActions({onCycleThroughAttachments}) {
+function CarouselActions({onCycleThroughAttachments}: CarouselActionsProps) {
     useEffect(() => {
         const shortcutLeftConfig = CONST.KEYBOARD_SHORTCUTS.ARROW_LEFT;
         const unsubscribeLeftKey = KeyboardShortcut.subscribe(
             shortcutLeftConfig.shortcutKey,
-            (e) => {
-                if (lodashGet(e, 'target.blur')) {
-                    // prevents focus from highlighting around the modal
-                    e.target.blur();
-                }
-
+            () => {
```

Comment:
e is KeyboardEvent which doesn't have 'target.blur'. Checked on every platform and on desktop/web target.blur is undefined and on mWeb/ios/android the event isn't triggered because obviously there's no keyboard 😄. **TL;DR this is removed because it was never used and it was giving problems with typing.**